requirements/pkgutils.txt

```diff
@@ -6,3 +6,4 @@ sphinx2rst>=1.0
 bumpversion==0.6.0
 pydocstyle==6.3.0
 mypy==1.14.1
+types-pycurl>=7.43.0.5; sys_platform != 'win32' and platform_python_implementation=="CPython"
```

Comment:
what is types-pycurl? did the package name changed?

Reply:
Was suggested by mypy. Otherwise mypy ci was failing

Reply:
Weren't we using only pycurl?

Reply:
I started to get this error only now, and types-pycurl was suggested. The error was related to mypy build (which I am not very familiar with). Then I saw other types-* are already used from the same repo. And then if pycurl is optional then full pycurl dependency is not required and types-pycurl would be sufficient to pass mypy build. I suppose mypy build does typechecking that is why it errored